packages/lesswrong/components/posts/dialogues/DialogueSubmit.tsx

```diff
@@ -73,6 +76,13 @@ const DialogueSubmit = ({
   const { captureEvent } = useTracking();
   if (!currentUser) throw Error("must be logged in to post")
 
+  const { create: createShortform, loading, error } = useCreate({
+    collectionName: "Comments",
+    fragmentName: 'CommentEdit',
+  })
+  const userShortfeed = currentUser?.shortformFeedId;
+  const [editors, _setEditors] = React.useContext(EditorsContext)
```

Comment:
why `_` prefix?

Reply:
unused. could just not match if you prefer

Reply:
oh, yeah, would prefer that